collect_app/src/main/java/org/odk/collect/android/widgets/VideoWidget.java

```diff
@@ -275,13 +215,6 @@ public void setBinaryData(Object object) {
         // Need to have this ugly code to account for
         // a bug in the Nexus 7 on 4.3 not returning the mediaUri in the data
         // of the intent - uri in this case is a file
```

Comment:
We also don't need this comment anymore.

Reply:
Thanks for directing @grzesiek2010!